services/dask-sidecar/src/simcore_service_dask_sidecar/settings.py

```diff
@@ -49,5 +49,5 @@ def as_worker(self) -> bool:
 
     @validator("LOG_LEVEL", pre=True)
     @classmethod
-    def _validate_loglevel(cls, value: Any) -> str:
-        return cls.validate_log_level(value)
+    def _validate_loglevel(cls, value: str) -> str:
```

Comment:
MINOR: Strictly speaking is a pre-validator and the value can actually be anything i.e. ``Any`` (even wrong type). The idea here is to "preprocess" the input value using the helper ``cls.validate_log_level``

Check the  pydantic's pre-validators (with ``pre=True``) in https://docs.pydantic.dev/usage/validators/#pre-and-per-item-validators

this should go in the line of

```python
@validator("LOG_LEVEL", pre=True)
@classmethod
def _validate_loglevel(cls, value: Any) -> str:
     return  cls.validate_log_level(str(value))
```

here ``str()`` will raise ``TypeError`` if is not a string. (note the validators should either return the parsed value or raise a ValueError, TypeError, or AssertionError (assert statements may be used) ) and ``cls.validate_log_level`` returns a string